extensions/macos-utilities/src/terminal-history.tsx

```diff
@@ -0,0 +1,133 @@
+import { ActionPanel, Action, Icon, Color, List, confirmAlert, Alert, showHUD } from "@raycast/api";
+import { useEffect, useState } from "react";
+import { execSync } from "child_process";
+import os from "os";
+import { createRaySoLink, getHistory, restoreHistory, saveFile } from "./utils";
+
+let history: string[] = [];
+
+export default function Command() {
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    setLoading(false);
+  }, [loading]);
+
+  history = getHistory();
+
+  const length = history.length;
+
+  if (length === 0) {
+    return (
+      <List>
+        <List.EmptyView
+          icon={{ source: "https://placekitten.com/500/500" }}
```

Comment:
Nit: I think the kitten is a placeholder from the examples, not actually intended for production. How about using `icon={Icon.Terminal}` instead?

https://developers.raycast.com/api-reference/user-interface/icons-and-images

Reply:
https://github.com/raycast/extensions/pull/9981/commits/6142c027e0c299f47f6554f6615583060a1b8628

Thanks for the suggestion, I have updated the icon to the following as suggested.